java/arcs/sdk/Handle.kt

```diff
@@ -42,25 +42,25 @@ interface ReadableCollection<T : Entity> : Handle {
     val size: Int
```

Comment:
size will need to change to be a function like size(), because you can't have suspend properties.